.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+
+---
+
+<!--
+YOUR ISSUE MAY BE CLOSED IF YOU DO NOT FOLLOW THIS TEMPLATE
+
+Consider searching for similar issues before submitting yours.
```

Comment:
Maybe include a link to the issues with the feature request tag

Reply:
Considering it couldn't be clicked anyways, I wasn't sure it was useful. I think it would be easier to just click on the issues button.

Reply:
Well, opening the issues takes a bit more work. You must filter by feature requests and include closed issues (since you'd be looking for examples). Whereas with a direct link the user just needs to copy and paste